app/src/main/res/layout/item_connector.xml

```diff
@@ -72,6 +72,7 @@
             android:layout_height="wrap_content"
             android:layout_marginTop="4dp"
             android:layout_marginBottom="8dp"
+            android:visibility="@{item.chargepoint.hasKnownPower() ? View.VISIBLE : View.GONE}"
```

Comment:
To simplify this (showing a view only when a boolean condition is true), I have defined a custom `BindingAdapter` that can be accessed with `app:goneUnless` (see `BindingAdapters.kt` under `net.vonforst.evmap.ui`).

Reply:
Ah, that makes sense. Fixed.